week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -31,13 +31,23 @@ var parkAvenueHouse = {
 */
 
 // returns the full name (first name + last name) of the owner of the house
-function getOwnerFullName(house) {}
+function getOwnerFullName(house) {
+  return house.currentOwner.firstName + " " + house.currentOwner.lastName;
+}
 
 // returns an array of the owners' email addresses of the two houses
-function getEmailAddresses(house1, house2) {}
+function getEmailAddresses(house1, house2) {
+  return house1.currentOwner.email + " , " + house2.currentOwner.email;

```

Comment:
Small note, this will return a string instead of an array. It prints the right thing but if you want to index into it you can't.